lms/templates/main.html

```diff
@@ -165,7 +165,9 @@
 </html>
 
 <%def name="login_query()">${
-  u"?next={0}".format(urlquote_plus(login_redirect_url)) if login_redirect_url else ""
+  u"?next={next}".format(
```

Comment:
Great.  Do we want a test around this somewhere so it doesn't vanish and it is clear why this was added?